docs/source/target-query.rst

```diff
@@ -0,0 +1,121 @@
+target-query
```

Comment:
I'm missing the target-query -s parameter to convert records to text. Really usefull if you just want to quickly grep through things and if you don't want to deal with rdump selectors.

Reply:
Thanks, added.